mesheryctl/internal/cli/root/app/onboard.go

```diff
@@ -41,6 +41,9 @@ var onboardCmd = &cobra.Command{
 		var err error
 		client := &http.Client{}
 
+		// get logger instance
+		log, _ := utils.MeshkitLogger()
```

Comment:
can we initialize logger instance in root.go, instead of each sub command ?

Reply:
I was thinking of the same initially but having a single instance will lead to passing that instance through many functions across different levels and will introduce unnecessary complexity in the flow, hence I went with initializing different instances something along the lines of what logrus provides